Swift/QtUI/QtXMLConsoleWidget.h

```diff
@@ -42,5 +42,7 @@ namespace Swift {
         private:
             QTextEdit* textEdit;
             QCheckBox* enabled;
+            QCheckBox* debugenabled;
```

Comment:
The value of these members is undefined until you assign a value to them. 
You can write `QCheckBox* debugEnabled = nullptr` in the class definition in the header file to do so. Furthermore it should be `debugEnabled` and not `debugenabled`, as we camelCase for member variables. See the Naming section in https://github.com/swift/swift/wiki/Coding-Style .